plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/server/behavior/OpenShiftLaunchController.java

```diff
@@ -73,8 +74,17 @@ protected static IControllableServerBehavior getServerBehavior(ILaunchConfigurat
 		IControllableServerBehavior behavior = server.getAdapter(IControllableServerBehavior.class);
 		return behavior;
 	}
-
 	
+	private String getLabel(String mode) {
+	    String label = OpenShiftCoreMessages.RunOnOpenshift;
+	    if (ILaunchManager.PROFILE_MODE.equals(mode)) {
```

Comment:
just a niggle: i think `switch-case` with "RunOnOpenshift" as the default would be more readable here